src/test/java/hw3/shape/Round.java

```diff
@@ -0,0 +1,22 @@
+package hw3.shape;
+
+public abstract class Round implements Shape {
+
+    private double bigRadius;
+    private double smallRadius;
+
+    public Round(double bigRadius, double smallRadius) {
+        this.bigRadius = bigRadius;
+        this.smallRadius = smallRadius;
+    }
+
+    public Round() {
+    }
+
+    @Override
```

Comment:
please remove this annotation from abstract classes, since you are not overriding it yet, but just implementing this method. you override it in your Circle class - there this usage is correct

Reply:
Annotation ware deleted from abstract classes Round + Rectangular